src/cpp/src/llm_pipeline.cpp

```diff
@@ -339,11 +394,354 @@ std::pair<std::string, Any> generation_config(const GenerationConfig& config) {
 }  // namespace genai
 }  // namespace ov
 
+namespace {
```

Comment:
Utils for `NPU` pipeline, perhaps the entire pipeline implementation with utils should be hidden in separate file

Reply:
Moved